docs/ReleaseInstructions.md

```diff
@@ -1,8 +1,8 @@
 # Release process
 
-### Prior to release: 1 week out
+## Prior to release: 1 week out
 
-* Ensure the version specified in `docs/config.ini`, `VERSION`, `docs/index.rst`, and `docs/badges.rst` align with the intended release version.
+* Ensure the version specified in `docs/conf.py`, `VERSION`, and `docs/index.md` align with the intended release version.
```

Comment:
No, we don't have/use the `docs/badges` file anymore on the new webpage. Additionally, I don't think it was used in the old webpage either, because basically the exact same lines that were in the `docs/badges.rst` file were also in the `docs/index.rst` file, and `docs/badges.rst` isn't imported anywhere in our code.